core/src/test/scala/org/apache/spark/metrics/sink/PrometheusServletSuite.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.metrics.sink
+
+import java.util.Properties
+
+import scala.collection.JavaConverters._
+
+import com.codahale.metrics.{Counter, Gauge, MetricRegistry}
+
+import org.scalatest.PrivateMethodTester
+import org.apache.spark.{SecurityManager, SparkConf, SparkFunSuite}
+
+
+class PrometheusServletSuite extends SparkFunSuite with PrivateMethodTester {
+
+  test("PrometheusServletSuite registered metrics test") {
```

Comment:
Instead of using test suite name, `PrometheusServletSuite`, shall we simple put "registered metrics" in the test name?